certified-connectors/LMS365/apiDefinition.swagger.json

```diff
@@ -1557,6 +1557,64 @@
                   },
                   "description": "Trainers"
                 },
+                "Admins": {
```

Comment:
@CJusjong Please create a new PR for `LMS365` as it is difficult to review multiple connectors in a single PR and the other connector have few issues need to be addressed. We can not proceed for production deployment for future updates of `LMS365` connector as the existing update is still not merged.